drivers/mpu9x50/Kconfig

```diff
@@ -5,30 +5,44 @@
 # directory for more details.
 #
 
-if TEST_KCONFIG
 
-choice
-    bool "MPU-9X50 accelerometer/magnetometer/gyroscope"
-    optional
+menuconfig MODULE_MPU9X50
+    bool
+    prompt "MPU-9X50 accelerometer/magnetometer/gyroscope" if !(MODULE_SAUL_DEFAULT && HAVE_MPU9X50)
+    default y if (MODULE_SAUL_DEFAULT && HAVE_MPU9X50)
+    depends on TEST_KCONFIG
     depends on HAS_PERIPH_I2C
-    help
-        This driver supports both MPU9150 and MPU9250. Choose one model.
+    select MODULE_PERIPH_I2C
+    select MODULE_ZTIMER
+    select MODULE_ZTIMER_MSEC
```

Comment:
I'd keep the `help` string.

Reply:
I added it back in but I somehow think it is equivalent to:
![image](https://user-images.githubusercontent.com/19396439/151961639-d4fda5ae-d74b-4ae9-91ef-a6ace6c4eda0.png)

Reply:
Think about generating documentation from Kconfig symbols in the future 😉